Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM python:3.5.6-alpine3.8
```

Comment:
let's use Python 3.6/3.7 image.

Reply:
airflow doesn't support py3.6 officially yet https://github.com/apache/incubator-airflow/pull/3816. Propose to add later when there will be official support.

Reply:
:)